src/components/loaders/littleLoader.tsx

```diff
@@ -61,15 +64,13 @@ const LoaderWrapper = styled.div<LoaderProps>`
     }
   }
 `
-export const LittleLoader: FC<LoaderProps> = ({ position }) => {
-  return (
-    <LoaderWrapper position={position}>
-      <div>
-        <div />
-        <div />
-        <div />
-        <div />
-      </div>
-    </LoaderWrapper>
-  )
-}
+export const LittleLoader: FC<LoaderProps> = ({ position, color }) => (
+  <LoaderWrapper position={position} color={color}>
+    <div>
+      <div />
+      <div />
+      <div />
+      <div />
+    </div>
```

Comment:
Becouse first `div` it`s wrapper which will be placed in page, it has width: 100%; height: 100% related on parent; and it place loader which inside to center because use flexBox; 

The Second is a loader wich has position: relative

And others divs inside Second have position: absolute